plugins/chunter-resources/src/components/chat-message/ChatMessageInput.svelte

```diff
@@ -107,21 +104,19 @@
 
   async function deleteTypingInfo (): Promise<void> {
     if (!withTypingInfo) return
-    const room = { objectId: object._id, objectClass: object._class }
-    updateMyPresence(room, { typing: undefined })
+    clearTyping(me)
   }
 
   async function updateTypingInfo (): Promise<void> {
     if (!withTypingInfo) return
 
     throttle.call(() => {
-      const room = { objectId: object._id, objectClass: object._class }
-      const typing = { person: me, lastTyping: Date.now() }
-      updateMyPresence(room, { typing })
+      setTyping(me)
```

Comment:
This is not correct, `objectId` is missing, this will show that you are typing in every channel.